orte/mca/ess/base/ess_base_std_prolog.c

```diff
@@ -56,6 +59,10 @@ int orte_ess_base_std_prolog(void)
             goto error;
         }
     }
+    /* check for OPAL_PREFIX */
+    if (NULL != (evar = getenv("OPAL_PREFIX"))) {
+        opal_setenv("PMIX_PREFIX", evar, false, &environ);
```

Comment:
@rhc54 while I'm checking if this fixes my particular case I have a coment.
This fix is focused on internal PMIx. This may break external pmix in following cases:
* if `OPAL_PREFIX` is set - we will set `PMIX_PREFIX` and this will mislead external pmix
* if no `OPAL_PREFIX` provided but user still needs to guide external PMIx - this fix will not help.
A;so